CLOUD/Get-GCPSizingInfo.ps1

```diff
@@ -163,3 +173,21 @@ Write-Host "Total capacity of all disks: $totalGiB GiB or $totalGB GB or $totalT
 Write-Host ""
 Write-Host "CSV file output to: $output" -foregroundcolor green
 $vmList | Export-CSV -path $output
+
+Write-Host
+Write-Host
+Write-Host "Results will be compressed into $archiveFile and original files will be removed." -ForegroundColor Green
+
+Stop-Transcript
+
+# Compress the files into a zip archive
+Compress-Archive -Path $outputFiles -DestinationPath $archiveFile
+
+# Remove the original files
+foreach ($file in $outputFiles) {
+    Remove-Item -Path $file -ErrorAction SilentlyContinue
+}
+
+Write-Host
+Write-Host
+Write-Host "Results have been compressed into $archiveFile and original files have been removed." -ForegroundColor Green
```

Comment:
@vpritamani-rubrik, please add an instructional line after this, telling the user to send the zip file to their Rubrik representative.